ingest_revisions_dataflow/dataflow_main.py

```diff
@@ -24,25 +33,16 @@ def run(argv=None):
                       dest='output',
                       # CHANGE 1/5: The Google Cloud Storage path is required
                       # for outputting the results.
-                      default='ingested/',
+                      default='gs://wikidetox-viz-dataflow/output_lists/ingestion_output.txt',
```

Comment:
I'd rather leave project names out of code, and have a JSON config file that is not checked in. This allows other to more naturally and easily re-run our work.